python/src/usecases/oscillator.py

```diff
@@ -0,0 +1,217 @@
+
+    muZetap : float
+          muZetap = 0.05, mean of the damping ratio of the primary system
+
+    sigmaOverMuZetap : float
+          sigmaOverMuZetap = 0.4, coefficient of variation of the damping ratio of the primary system
+
+    distributionZetap : :class:`~openturns.LogNormal` distribution of the damping ratio of the primary system
+          distributionZetap = ot.LogNormalMuSigmaOverMu(muZetap, sigmaOverMuZetap).getDistribution()
+
+    muZetas : float
+          muZetas = 0.02, mean of the damping ratio of the secondary system
+
+    sigmaOverMuZetas : float
+          sigmaOverMuZetas = 0.5, coefficient of variation of the damping ratio of the secondary system
+
+    distributionZetas : :class:`~openturns.LogNormal` distribution of the damping ratio of the secondary system
+          distributionZetas = ot.LogNormalMuSigmaOverMu(muZetas, sigmaOverMuZetas).getDistribution()
+
+    muFs : float
+          muFs = 27.5, mean of the force capacity of the secondary spring
+
+    sigmaOverFs : float
+          sigmaOverFs = 0.1, coefficient of variation of the force capacity of the secondary spring
+
+    distributionFs : :class:`~openturns.LogNormal` distribution of the force capacity of the secondary spring
+          distributionFs = ot.LogNormalMuSigmaOverMu(muFs, sigmaOverFs).getDistribution()
+
+    muS0 : float
+          muS0 = 100, mean of the intensity of the white noise
+
+    sigmaOverS0 : float
+          sigmaOverS0 = 0.1, coefficient of variation of the intensity of the white noise
+
+    distributionS0 : :class:`~openturns.LogNormal` distribution of the intensity of the white noise
+          distributionS0 = ot.LogNormalMuSigmaOverMu(muS0, sigmaOverS0).getDistribution()
+
+    distribution : :class:`~openturns.ComposedDistribution`
+                   The joint distribution of the input parameters
+
+    Examples
+    --------
+    >>> from openturns.usecases import oscillator
+    >>> # Load the oscillator
+    >>> osc = oscillator.Oscillator()
+    """
+
+    def __init__(self):
+        self.dim = 8
+
+        # Random variable : mp
+        self.muMp = 1.5
+        self.sigmaOverMuMp = 0.1
+
+        # Random variable : ms
+        self.muMs = 0.01
+        self.sigmaOverMuMs = 0.1
+
+        # Random variable : kp
+        self.muKp = 1.0
+        self.sigmaOverMuKp = 0.2
+
+        # Random variable : ks
+        self.muKs = 0.01
+        self.sigmaOverMuKs = 0.2
+
+        # Random variable : zetap
+        self.muZetap = 0.05
+        self.sigmaOverMuZetap = 0.4
+
+        # Random variable : zetas
+        self.muZetas = 0.02
+        self.sigmaOverMuZetas = 0.5
+
+        # Random variable : Fs
+        self.muFs = 27.5
+        self.sigmaOverMuFs = 0.1
+
+        # Random variable : S0
+        self.muS0 = 100.0
+        self.sigmaOverMuS0 = 0.1
+
+        # create the limit state function model
+        formula = "Fs-3*ks*sqrt("
```

Comment:
Please separate arithmetic operators with blank spaces so that the code can be read more easily.

Reply:
Actually, black is not able to modify strings formula.

Reply:
Oh, nevermind then :)